corehq/apps/userreports/models.py

```diff
@@ -151,9 +151,17 @@ def indicators(self):
                 }
             }
         }, self.named_filter_objects)
+        repeat_iteration_indicator = IndicatorFactory.from_spec({
+            "column_id": "repeat_iteration",
+            "type": "repeat_iteration",
+            "display_name": "base document iteration",
+            "datatype": "integer",
+            "is_nullable": False,
+            "is_primary_key": True,
+        }, self.named_filter_objects)
```

Comment:
All tables include a `repeat_iteration` column. This value will be `0` for documents that aren't part of a repeat.